ziti/cmd/create/config_templates/controller.yml

```diff
@@ -29,6 +29,8 @@ identity:
   {{ if not .Controller.Web.Identity.AltCertsEnabled }}#{{ end }}  - server_cert:  "{{ .Controller.Web.Identity.AltServerCert }}"
   {{ if not .Controller.Web.Identity.AltCertsEnabled }}#{{ end }}    server_key:   "{{ .Controller.Web.Identity.AltServerKey }}"
 
+trustDomain: quickstart
```

Comment:
this one i'd like to change to an environment variable. i'll offer you a patch to apply if that's ok?

Reply:
Lets do it in another PR.